src/Donations/PaymentMethods/PaymentFunctions.ts

```diff
@@ -170,7 +170,7 @@ export function createDonationData({
         : amount,
     currency,
     donor: { ...contactDetails },
-    frequency: frequency === "once" ? null : frequency,
+    frequency: frequency,
```

Comment:
This does not seem to belong to the locale change - is it related to the bug @sagararyal mentioned at https://planetde.slack.com/archives/C02393H3GDC/p1643748738179739?